external-plugins/rehearse/Makefile

```diff
@@ -10,7 +10,7 @@ format:
 	gofmt -w .
 
 test:
-	$(bazelbin) test //external-plugins/rehearse/plugin:*
+	$(bazelbin) test //external-plugins/rehearse/...
```

Comment:
Fixed to allow running all tests inside rehearse plugin folder.

Reply:
+1